### PR TITLE
dashboard url in home

### PR DESCRIPTION
**Pull request by tanvirstreame**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/tareqmonwer/.local/share/virtualenvs/Django-School-Management-Bw2s9AGf/bin/python"
+}
```
Comment (tanvirstreame) [inline .vscode/settings.json]: What do these do? I think it is your local setup. Remove it.

File: Pipfile [hunk 2]
```diff
@@ -5,6 +5,7 @@ verify_ssl = true
 
 [dev-packages]
 autopep8 = "*"
+pylint = "*"
```
Comment (tanvirstreame) [inline Pipfile]: what is this?

File: .vscode/settings.json (hunk 1 above)
Comment (TareqMonwer) [reply .vscode/settings.json]: okay.

File: Pipfile (hunk 2 above)
Comment (TareqMonwer) [reply Pipfile]: It's from local settings. Removing now.

**Pull request merged**